pulp_container/app/tasks/recursive_add.py

```diff
@@ -57,12 +58,13 @@ def recursive_add_content(repository_pk, content_units):
         )
     )
 
-    blobs_to_add = (
-        Blob.objects.filter(pk__in=content_units)
-        | Blob.objects.filter(pk__in=manifests_to_add.values_list("blobs", flat=True))
-        | Blob.objects.filter(pk__in=manifests_to_add.values_list("config_blob", flat=True))
+    blobs_to_add = Blob.objects.filter(pk__in=content_units) | Blob.objects.filter(
+        pk__in=manifests_to_add.values_list("blobs", flat=True)
     )
 
+    config_blobs_to_add = ConfigBlob.objects.filter(
+        pk__in=manifests_to_add.values_list("config", flat=True)
+    )
```

Comment:
Will the copy/remove workflow work for config blobs that have not been migrated yet? I think not, but this might be a corner case we may want to support for a couple of releases. We will want to have `Blob.objects.filter(pk__in=manifests_to_add.values_list("config_blob", flat=True))` included as well. Do you agree?